packages/sources/css/src/components/navigation/tabs/src/index.css

```diff
@@ -9,24 +9,21 @@
   block-size: fit-content;
   inline-size: 100%;
   background-color: var(--vtmn-semantic-color_background-primary);
-}
-
-.vtmn-tabs > ul {
   display: block;
   white-space: nowrap;
   overflow-x: auto;
   margin: 0;
   padding: 0;
 }
 
-.vtmn-tabs li {
+.vtmn-tabs > li {
   display: inline-flex;
   padding-inline: rem(2px);
   padding-block-start: rem(2px);
 }
 
-.vtmn-tabs a,
-.vtmn-tabs_size--medium a {
+.vtmn-tabs > li > button,
+.vtmn-tabs_size--medium a > li > button {
```

Comment:
I think you forgot to remove the `a` in the selector.

Reply:
Indeed I already mention similar issues on this style. More globally, the CSS should not make assumptions on the HTML structure. But it's a minor change for now, I think the whole style would be reworked for the v1 :)